### PR TITLE
SUM-254 - Heading Size to cards

### PR DESCRIPTION
**Pull request by mdyoung3**
# READY FOR REVIEW

# Summary
- Adds Heading Level via paragraph behaviors for accessibility and Visual improvements.
- Ticket:  https://stanfordits.atlassian.net/browse/SUM-254

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by mdyoung3** [comment]
@pookmish Did you get a chance to look at this? I believe it's a blocker for Rebecca

File: docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php [hunk 1]
```diff
@@ -31,6 +31,19 @@ class SummerCardBehaviors extends CardBehavior {
    */
   public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, FormStateInterface $form_state): array {
     $element = parent::buildBehaviorForm($paragraph, $form, $form_state);
+    $element['sum_card_heading_size'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Heading Size'),
+      '#description' => $this->t('Adjust header size for accessibility and visual improvements.'),
+      '#weight' => 0,
+      '#empty_option' => $this->t('Default'),
```
Comment (pookmish) [inline docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php:45]: What is "Default" compared to the 3 options?

**Review by pookmish — CHANGES_REQUESTED** [review]
There's a merge conflict too. can you review

File: docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php (hunk 1 above)
Comment (mdyoung3) [reply docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php:45]: Per the ticket, "uses the Heading Level size."

Maybe @rebeccahongsf can fill in, but it seems like the default applies no alterations.

Comment (mdyoung3) [reply docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php:45]: Whereas the other options change the font-size.

Comment (mdyoung3) [reply docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php:45]: @pookmish Conflict resolved.

Comment (pookmish) [reply docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php:45]: I'm just looking at it from a content editor perspective. I have no idea if "Default" is larger or smaller than any of the 3 choices.

Comment (rebeccahongsf) [reply docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php:45]: Cross posting from JIRA @mdyoung3 @pookmish 😎 

That’s a great point and I had the same thoughts as Marc — I assumed that Default would use the default size coming from the Heading Level. The editor would leave the option as-is if they are content with the default typography size from the Heading Level option.

For additional context, I received feedback from Kat and Kerri on adding the Heading Size option was to maintain accessible headings while having them reasonably sized for longer quotes/heading content. 

Feel free to tag me again if there are any additional Qs or concerns!

Comment (pookmish) [reply docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php:45]: Let's try to keep the conversation in one or the other locations. Please see Jira for my comment.

**Review by pookmish — APPROVED** [review]

**Pull request merged**